layers/config/overrides.js

```diff
@@ -137,6 +137,14 @@ BR.confLayers.getPropertyOverrides = function() {
             'mapUrl': 'https://osm.ch/#{zoom}/{lat}/{lon}',
             'worldTiles': true
         },
+        'swisstopo-landeskarte': {
+            'country_code': 'CH',
+            'mapUrl': 'https://wmts.geo.admin.ch/1.0.0/ch.swisstopo.pixelkarte-farbe/default/current/3857/{z}/{x}/{y}.jpeg',
```

Comment:
`mapUrl` is not the tile url, but a link to the original map that is shown for the active layers above the attribution on the bottom right.

In this case that would probably be https://map.geo.admin.ch/. The current map view can be passed with `{zoom}/{lat}/{lon}` placeholders, but I don't know if that map understands lat/lon.

Reply:
The map itself certainly understands lat/lon, but not sure it can be adressed like that by URL. By default it works with the "CH1903 LV95" coordinate system. By replacing the &E= and &N= heading by &lat= &lon= it switches to 3D mode.

Reply:
According to the [help section](https://help.geo.admin.ch/?ids=54&lang=de), we can use `swisssearch`:

https://map.geo.admin.ch/?swisssearch=47.378319,8.542399

It seems that the `zoom` parameter is ignored though, when using `swisssearch`.

Will update the URL, thank you for the hints!